apps/whale-api/docker-compose.yml

```
@@ -2,7 +2,7 @@ version: '3.7'
 
 services:
   defi-blockchain:
-    image: defi/defichain:master-c14a2ba13
+    image: defi/defichain:master-5a4b409ac
```

```suggestion
    image: defi/defichain:master-8594d0aec
```

To include some fixes that were added